ci_feature/Dockerfile

```diff
@@ -5,6 +5,7 @@ LABEL vendor=Microsoft\ Corp \
     com.microsoft.version="6.0.0-1"
 ENV tmpdir /opt
 ENV APPLICATIONINSIGHTS_AUTH OTQzNWI0M2YtOTdkNS00ZGVkLThkOTAtYjA0Nzk1OGU2ZTg3
+ENV APPLICATIONINSIGHTS_ENDPOINT https://dc.services.visualstudio.com/v2/track
```

Comment:
Can we have a config map since the yaml is anyways different between public cloud, MC and Fairfax, and inject the endpoint through config map ?
Where will this value be set for MC, Fairfax and Blackforest(when we go live there)?